lib/unzip/index.js

```diff
@@ -0,0 +1,27 @@
+const unzip = require('unzip2');
+const { entryTypes } = require('../constants');
+const detectTranslationsFileType = require('../utils/detectFileType');
+const textParser = require('../parser');
+
+const unzipTranslationsFromStream = (stream, callback) => stream
+  .pipe(unzip.Parse())
+  .on('entry', (entry) => {
+    const { type, path } = entry;
+    const translationsFileType = detectTranslationsFileType(path);
+
+    if (type === entryTypes.FILE && !!translationsFileType) {
+      entry.on('data', (data) => {
+        const fileText = data.toString();
+        textParser(fileText, translationsFileType)
```

Comment:
unzip should contain only related thing to unzip
`textParser` is not part of unzip.
Event we would need to parser part as separate public methods (both files to data and data to files)

Reply:
This part not finished yet. It only for checking etc.